src/cheri/insns/gcperm_32bit.adoc

```diff
@@ -19,22 +19,14 @@ Encoding::
 include::wavedrom/gcperm.adoc[]
 
 Description::
-If MXLEN=32 unpack the <<AP-field>> from the format in <<cap_perms_encoding32>>.
-+
-Convert the unpacked <<AP-field>> as well as the <<SDP-field>> and <<section_cap_level, CL-field>> of capability `cs1`
-into a bit field, with the format shown in xref:gcperm_bit_field[xrefstyle=short],
+Convert the unpacked <<AP-field>>, <<SDP-field>>, and any other extension-defined permission-like fields of capability `cs1`
+into a bit field, with the same format as used by <<ACPERM>> (see <<acperm_bit_field>>),
 and write the result to `rd`.
++
 All bits in the `[23:0]` range that are reserved or assigned to extensions that are not implemented by the current hart always report 1.
 +
 All architectural permission bits in `rd` are set to 0 if any <<section_cap_integrity,integrity>> checks failed.
 +
-.Capability permissions bit field
-[#gcperm_bit_field]
```

Comment:
I know this is redundant, but I do kind of like seeing the bitfield values in both instructions. Don't really mind either way.